source/compute/software/r_package_management.rst

```diff
@@ -122,7 +162,8 @@ Installing R packages using conda
     Usage of conda is discouraged in the clusters at UAntwerpen, UGent,
     and VUB.
 
-The easiest way to install and manage your own R environment(s) is conda.
+The paragraphs below illustrate how to install and use R and R packages
```

Comment:
Two more requests:
- please check that everywhere, Conda is mentioned with capital-C (official name)
- starting from line 187, I propose the following change for better formatting:

If the result is blank, or reports that Conda can not be found, modify the ``PATH`` environment variable appropriately by adding miniconda’s ``bin`` directory to ``PATH``.